backend/src/main/java/com/benine/backend/http/EditPresetHandler.java

```diff
@@ -92,7 +96,8 @@ public void updatePosition(String camID, List<String> tagList, int presetID) thr
   MalformedURIException {
     IPCamera ipcam = (IPCamera)control.getCameraController()
         .getCameraById(Integer.parseInt(camID));   
-    Preset newPreset = CreatePresetHandler.setPreset(ipcam, tagList);
+    CreatePresetHandler handler = new CreatePresetHandler();
```

Comment:
I don't think this is behavior we want. The responsibility of the `CreatePresetHandler` is handling the endpoint, not setting presets.